tests/e2e/vsan_test.go

```diff
@@ -63,6 +64,10 @@ func (s *VsanTestSuite) SetUpTest(c *C) {
 }
 
 func (s *VsanTestSuite) TearDownTest(c *C) {
+	if len(s.containerList) > 0 {
+		out, err := dockercli.RemoveAllContainers(s.config.DockerHosts[0])
```

Comment:
@lipingxue By looking at your test code I don't see a need for cleaning up containers. Let's remove it.